hudi-utilities/src/test/resources/delta-streamer-config/sql-file-transformer.sql

```diff
@@ -0,0 +1,27 @@
+
+--  Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
why the license is this way? Usually its like below
```
/*
 *
```

Reply:
For SQL files this is the supported comment style, when I use the block comment the test throws error.